sidecar/Dockerfile

```
@@ -55,6 +55,9 @@ COPY --chown=root:root files/ss-xroad.conf /etc/supervisor/conf.d/xroad.conf
 COPY --chown=xroad:xroad files/local.conf /etc/xroad/services/local.conf
 COPY --chown=xroad:xroad files/*logback* /etc/xroad/conf.d/
 COPY --chown=xroad:xroad files/custom-fetch-pin.sh /usr/share/xroad/autologin/
+
+RUN sed -i 's/initctl/supervisorctl/g' /usr/share/xroad/scripts/_restore_xroad.sh
```

Note. Modifying the script with sed is a bit brittle; if the script changes so that the `initctl` string is no longer present, this fails silently (or might fail to do the right thing since it matches also substrings). 
It is probably OK here since we plan to incorporate the change in the original script, but something to keep in mind.

---

The first thing I had done was replace the whole file but this also has the problem that the changes would be lost if the file is updated, I don't know what is the best option

---

Since this should be a temporary fix, let's keep it simple.